app/controllers/packages_controller.rb

```diff
@@ -1,19 +1,36 @@
 class PackagesController < ApplicationController
   before_action :set_event, only: [:index, :show]
   before_action :set_package, only: [:show]
+  before_action :set_photographers_present, only: [:index, :show]
 
-  def index
+   def index
     @packages = Package.all
+    @event_time_slots = (1..12).to_a
+    @unavailable_times = []
+    @event_time_slots.each do |event_time_slot|
+      @time_slot_available = []
```

Comment:
Same for this one, I think it could be a local variable.

Reply:
@yassinmelsir @Matth93 let me know what you guys decide here.  I want to make sure I'm clear on which is better practice for this scenario

Reply:
It's a local variable now :) @elbyrd09 @Matth93

Reply:
time_slot_available and event_time_slots that is